src/main/java/fr/uge/dummychargepoint/socket/ConfigurationApplication.java

```diff
@@ -0,0 +1,13 @@
+package fr.uge.dummychargepoint.socket;
+
+public interface ConfigurationApplication {
+
+  default void start() {
+    throw new IllegalCallerException("Client cannot start a websocket connection");
```

Comment:
bad practice, if the default method is an error due to a bad responsibility given to an implementation, you are breaking the Interface Segregation Principle

Reply:
So the error should be done on the implementation side ?
What about some method of Map, etc ? They do the same as here no ?

But I can move this logic into the correct implementation